playwright/fixtures/accounts.ts

```diff
@@ -17,47 +16,47 @@ const shortenAddress = (address = '', maxLength = 10, leftSlice = 5, rightSlice
   return `${address.slice(0, leftSlice)}...${address.slice(-rightSlice)}`
 }
 
+export type Dependencies = {
```

Comment:
no need to depend on a provider, use `testClient` from test contracts file directly